app/src/androidTest/java/com/duckduckgo/app/browser/BrowserWebViewClientTest.kt

```diff
@@ -90,6 +92,26 @@ class BrowserWebViewClientTest {
         verify(listener).requiresAuthentication(authenticationRequest)
     }
 
+    @Test
+    fun whenRenderProcessGoneDueToCrashThenCrashDataStoreEntryIsIncremented() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
I *thought* there was an annotation to specify the min test API but I couldn't find one so I went with this less elegant solution. Happy to update if you can think of it.

Reply:
`@SdkSuppress(minSdkVersion = Build.VERSION_CODES.O)` I think would do it

Reply:
Thanks the one!! Thanks :)